Packs/CommonTypes/Layouts/layoutscontainer-Malware.json

```
@@ -0,0 +1,172 @@
+{
+    "group": "indicator",
+    "id": "Malware",
```

```suggestion
    "id": "Malware V2",
```
This should solve the conflict with malware incident type layout